src/styles/main.scss

```diff
@@ -1,3 +1,2 @@
-body {
-  margin: 0;
-}
+@import "./blocks/body.scss";
+@import "./blocks/calendar.scss";
```

Comment:
i cannot find this file
<img width="364" alt="image" src="https://github.com/mate-academy/layout_calendar/assets/70220667/d24eeedb-4a27-4b1c-a040-417d7bfebeb0">
with calendar styles